components/uniset/src/enum_prop_mapping.rs

```diff
@@ -0,0 +1,862 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Didn't we decide to move this file into provider_ppucd until we need it outside?

Reply:
Done. Yes, I forgot about that, thanks.